CRM/Contribute/Form/Contribution.php

```diff
@@ -377,10 +377,6 @@ public function setDefaultValues() {
       $defaults['fee_amount'] = CRM_Utils_Money::format($defaults['fee_amount'], NULL, '%a');
     }
 
-    if (isset($defaults['net_amount'])) {
```

Comment:
We should be setting this to correct value rather than deliberately introducing this bug to solve a higher priority one. @pradpnayak please provide suggestion here for correct calculation.